### PR TITLE
Fix ordering of code fixes

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://devdiv.visualstudio.com/DevDiv/_queries/edit/1599579

I broke this in https://github.com/dotnet/roslyn/pull/59582.  Working on test now.

### CODE REVIEW

**Review by ryzngard — APPROVED** [review]
signoff pending test addition

**Comment by CyrusNajmabadi** [comment]
@ryzngard Test added.

**Review by ryzngard — APPROVED** [review]

File: src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs [hunk 1]
```diff
@@ -107,5 +109,53 @@ public void TestNoCyclesInRefactoringProviders()
...
+        public void TestCodeFixServiceOrderIsCorrect()
+        {
+            // This test will fail if a cycle is detected in the ordering of our code fix providers.
+            // If this test fails, you can break the cycle by inspecting and fixing up the contents of
+            // any [ExtensionOrder()] attributes present on our code fix providers.
+            var providers = ExportProvider.GetExports<CodeFixProvider, CodeChangeProviderMetadata>();
+            var providersPerLanguage = providers.ToPerLanguageMapWithMultipleLanguages();
+
+            TestCodeFixServiceOrderIsCorrectForLanguage(LanguageNames.CSharp, providersPerLanguage);
+            TestCodeFixServiceOrderIsCorrectForLanguage(LanguageNames.VisualBasic, providersPerLanguage);
```
Comment (ryzngard) [inline src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs:123]: ❓is `InlineData` better here so it's a single test failure per language?

Comment (CyrusNajmabadi) [reply src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs:123]: sure :)

Comment (CyrusNajmabadi) [reply src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs]: yup.  done.

**Pull request merged**